Tests/DNSClientTests/DNSTCPClientTests.swift

```diff
@@ -40,75 +56,82 @@ final class DNSTCPClientTests: XCTestCase {
+        }
     }
     
     func testSendTxtQuery() throws {
-        let result = try dnsClient.sendQuery(forHost: "google.com", type: .txt).wait()
-        XCTAssertGreaterThanOrEqual(result.header.answerCount, 1, "The returned answers should be greater than or equal to 1")
+        try testClient { dnsClient in
+            let result = try dnsClient.sendQuery(forHost: "google.com", type: .txt).wait()
+            XCTAssertGreaterThanOrEqual(result.header.answerCount, 1, "The returned answers should be greater than or equal to 1")
+        }
     }
     
     func testSendQueryMX() throws {
-        let result = try dnsClient.sendQuery(forHost: "gmail.com", type: .mx).wait()
-        XCTAssertGreaterThanOrEqual(result.header.answerCount, 1, "The returned answers should be greater than or equal to 1")
+        try testClient { dnsClient in
+            let result = try dnsClient.sendQuery(forHost: "gmail.com", type: .mx).wait()
+            XCTAssertGreaterThanOrEqual(result.header.answerCount, 1, "The returned answers should be greater than or equal to 1")
+        }
     }
 
     func testSendQueryCNAME() throws {
-        let result = try dnsClient.sendQuery(forHost: "www.youtube.com", type: .cName).wait()
-        XCTAssertGreaterThanOrEqual(result.header.answerCount, 1, "The returned answers should be greater than or equal to 1")
+        try testClient { dnsClient in
+            let result = try dnsClient.sendQuery(forHost: "www.youtube.com", type: .cName).wait()
+            XCTAssertGreaterThanOrEqual(result.header.answerCount, 1, "The returned answers should be greater than or equal to 1")
+        }
     }
 
     func testSRVRecords() throws {
-        let answers = try dnsClient.getSRVRecords(from: "_mongodb._tcp.ok0-xkvc1.mongodb.net").wait()
-        XCTAssertGreaterThanOrEqual(answers.count, 1, "The returned answers should be greater than or equal to 1")
+        try testClient { dnsClient in
+            let answers = try dnsClient.getSRVRecords(from: "_caldavs._tcp.google.com").wait()
+            XCTAssertGreaterThanOrEqual(answers.count, 1, "The returned answers should be greater than or equal to 1")
+        }
     }
     
     func testSRVRecordsAsyncRequest() throws {
-        let expectation = self.expectation(description: "getSRVRecords")
-        
-        dnsClient.getSRVRecords(from: "_mongodb._tcp.ok0-xkvc1.mongodb.net")
-            .whenComplete { (result) in
-                switch result {
-                case .failure(let error):
-                    XCTFail("\(error)")
-                case .success(let answers):
-                    print(answers)
-                    XCTAssertGreaterThanOrEqual(answers.count, 1, "The returned answers should be greater than or equal to 1")
+        testClient { dnsClient in
+            let expectation = self.expectation(description: "getSRVRecords")
+            
+            dnsClient.getSRVRecords(from: "_caldavs._tcp.google.com")
```

Comment:
Any reason for the change of address

Reply:
Because Google's caldav is unlikely to remove their SRV records. And my MongoDB server apparently just did, so it wasn't as stable as I thought it to be.